src/python/grpcio/grpc/_runtime_protos.py

```diff
@@ -15,6 +15,48 @@
 import sys
 
 _REQUIRED_SYMBOLS = ("_protos", "_services", "_protos_and_services")
+_MINIMUM_VERSION = (3, 5, 0)
```

Comment:
In future, we could consider adding type annotations to this file.

Reply:
[Issue created to track this cleanup.](https://github.com/grpc/grpc/issues/25006)